nncf/common/graph/graph_matching.py

```diff
@@ -61,18 +69,26 @@ def _find_all_matches(self, nodes, graph):
                 all_matches.append((n, following))
         return all_matches
 
-    def all_matches(self, nodes, graph):
+    def all_matches(self, nodes: List[Dict], graph: nx.DiGraph):
         return self._find_all_matches(nodes, graph)
 
-    def match(self, nodes, graph):
+    def match(self, nodes: List[Dict], graph: nx.DiGraph):
         all_matches = self._find_all_matches(nodes, graph)
         if not all_matches:
             return None, None
         return max(all_matches, key=lambda x: len(x[0]))
 
 
 class ConcatExpression(Expression):
-    def __init__(self, expressions):
+    """
+    A composite expression that matches to a node path in the graph which is a concatenation
+    of the paths that each matches, in order, to the sub-expression of this expression.
+    E.g. a subgraph (conv2d) -> (batch_norm) will be matched to the expression:
+        ConcatExpression([NodeExpression('conv2d'), NodeExpression('batch_norm2d')])
+    or, equivalently, using the overloaded operator syntax:
+        NodeExpression('conv2d') + NodeExpression('batch_norm2d')
+    """
+    def __init__(self, expressions: List[Expression]):
```

Comment:
```suggestion
    """
    
    def __init__(self, expressions: List[Expression]):
```

Please, use the following format of the docstring:
```
class ModelTransformer:
    """
    Applies transformations to the model.

    :param public_attribute: Public attribute description
    """

    def __init__(self, model: ModelType, transformation_layout: TransformationLayout):
        """
        Initializes Model Transformer

        :param model: The model to be transformed
        :param transformation_layout: An instance of `TransformationLayout` that
            includes a list of transformations to be applied to the model.
        """
        self._model = model
        self._transformations = transformation_layout.transformations
        self.public_attribute = None

    def transform(self) -> ModelType:
        """
        Applies transformations to the model.

        :return: The transformed model
        """
        raise NotImplementedError()
```

Reply:
Done